pyibisami/run_tests.py

```diff
@@ -170,15 +177,19 @@ def run_tests(**kwargs):
                         "plot_names": plot_names,
                         "description": description,
                         "plot_colors": colors,
-                        "ref_dir": str(ref_dir),
+                        "ref_dir": ref_dir,
                     },
                 )
                 try:
+                    cwd = Path().cwd()
+                    chdir(out_dir)  # So that the images are saved in the output directory.
```

Comment:
Even though this diff isn't showing it.  This is how you were already doing it.  I've just updated to use a pathlib object instead of an os path string.

Original
```
                    tmp_dir = os.getcwd()
                    os.chdir(out_dir)
                    interpreter.file(open(op.join(test_dir, test + '.em')))
                    os.chdir(tmp_dir)
```

Reply:
Ah, got it. Thanks.